vulcanai2/datasets/tabulardataset.py

```diff
@@ -282,35 +314,56 @@ def reverse_create_all_one_hot_encodings(self, prefix_sep="@"):
             max_columns = dummy_df.idxmax(axis=1)
 
             # Remove category_ prefix
-            result_series[dummy] = max_columns.apply(lambda item: item.split(prefix_sep)[1])
+            result_series[dummy] = max_columns.apply(lambda item:
+                                                     item.split(prefix_sep)[1])
 
         # Copy non-dummy columns over.
         for col in non_dummy_cols:
             result_series[col] = self.df[col]
 
         self.df = pd.DataFrame(result_series)
 
-        logger.info(f"Successfully converted {len(dummy_tuples)} columns back from dummy format.")
+        logger.info(f"Successfully converted {len(dummy_tuples)} \
+                    columns back from dummy format.")
 
     def identify_sufficient_non_null(self, threshold):
         """
-        Return columns where there is at least threshold percent of non-null values.
-        Does not drop columns
-        :param threshold: A number between 0 and 1, representing proportion needed
-        :return: A list of those columns with threshold percentage null values
+        Return columns where there is at least threshold percent of
+        non-null values.
+
+        Parameters
+        ----------
+        threshold: Float
+            A number between 0 and 1, representing proportion of non-null.
+
+        Returns
+        -------
+        cols: List
+            A list of those columns with threshold percentage null values
         """
+
         if threshold >= 1 or threshold <= 0:
-            raise ValueError("Threshold needs to be a proportion between 0 and 1")
+            raise ValueError(
+                "Threshold needs to be a proportion between 0 and 1")
         num_threshold = (threshold * len(self))
-        tmp = self.df.dropna(thresh=num_threshold, axis=1)  # thresh is "Require that many non-NA values."
+        # thresh is "Require that many non-NA values."
+        tmp = self.df.dropna(thresh=num_threshold, axis=1)
```

Comment:
are you sure `dropna` isn't done inline?

Reply:
@rfratila default is false for dropna

Reply:
ok as long as self.df isn't changed this is fine.

Reply:
I tested and can confirm it's not done inline